### ISSUE TITLE
Failed to Start - No PID

### ISSUE THREAD

**Issue by VAVANO** [devryan/GamePanelX-V3#175]
Hello, is there any Solution for this issue posted here, please?

![image](https://user-images.githubusercontent.com/31014506/29287689-9f914002-8136-11e7-8921-b63337eedfda.png)

Creating Templates and installing Server is all without any issues..

**Comment by will2therich** [comment]
Hi @VAVANO 
In a SSH session to the server can you start a screen session by running the command "screen"?
Ta

**Comment by VAVANO** [comment]
Yes, i have a screenshot here. i made just "screen" into console to see what happens: http://prntscr.com/gfrzz7
